.github/workflows/docker-unified.yml

```diff
@@ -415,6 +415,14 @@ jobs:
   smoke_test:
     name: Run Smoke Tests
     runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        test_strategy: [
+          "no_cypress", 
```

Comment:
So basically this controls which subset of all smoke tests we run? We can parallelize cypress and the non-cypress tests?

Reply:
we can parallelize non-cypress as well as within cypress tests. So this config and some changes in `integration_test` file which defines what cypress tests are in which suite.